.pre-commit-config.yaml

```diff
@@ -1,28 +0,0 @@
-- repo: https://github.com/pre-commit/pre-commit-hooks.git
```

Comment:
I like to use these hooks http://pre-commit.com/

Enforce me to keep the conventions. Do You have alternatives?